TickSpec/FeatureParser.fs

```diff
@@ -49,19 +49,21 @@ let parseFeature (lines:string[]) =
 
         examples
         |> Seq.map (fun exampleBlock ->
-            exampleBlock.Table.Header |> List.sort,
-            exampleBlock.Table.Rows
-            |> Seq.map (fun row ->
-                let rowMap =
-                    Seq.zip exampleBlock.Table.Header row
-                    |> Seq.fold (fun map (header, value) ->
-                            match Map.tryFind header map with
-                            | Some x ->
-                                let m = sprintf "A single header was specified multiple times in an example block starting at row %d" exampleBlock.LineNumber
-                                ParseException(m, Some exampleBlock.LineNumber) |> raise
-                            | None -> map |> Map.add header value)
-                        Map.empty
-                rowMap, exampleBlock.Tags))
+            let headers = exampleBlock.Table.Header |> List.sort
+            let rows =
+                exampleBlock.Table.Rows
+                |> Seq.map (fun row ->
+                    let rowMap =
+                        Seq.zip exampleBlock.Table.Header row
+                        |> Seq.fold (fun map (header, value) ->
+                                match Map.tryFind header map with
+                                | Some x ->
+                                    let m = sprintf "A single header was specified multiple times in an example block starting at row %d" exampleBlock.LineNumber
+                                    ParseException(m, Some exampleBlock.LineNumber) |> raise
+                                | None -> map |> Map.add header value)
+                            Map.empty
+                    rowMap, exampleBlock.Tags)
+            (headers, rows))
```

Comment:
move L64 to L57 using `||>`, remove parens from 66

Reply:
Changed, thanks.